custom_metrics/responsive_images.js

```diff
@@ -0,0 +1,959 @@
+[responsive-images]
```

Comment:
Comment this out when committing. Uncomment locally for testing.
```suggestion
//[responsive-images]
```